boefjes/boefjes/docker_boefjes_runner.py

```diff
@@ -53,16 +53,22 @@ def run(self) -> None:
 
             # save container log (stderr) to bytes
             self.bytes_api_client.login()
-            self.bytes_api_client.save_raw(task_id, container_logs, stderr_mime_types)
+            task = self.scheduler_client.get_task(task_id)
+
+            if task.status == TaskStatus.RUNNING:
+                self.boefje_meta.ended_at = datetime.now(timezone.utc)
+                self.bytes_api_client.save_boefje_meta(self.boefje_meta)  # The task didn't create a boefje_meta object
+
+            self.bytes_api_client.save_raw(task_id, container_logs, stderr_mime_types.union({"stderr/boefje"}))
```

Comment:
It was already in the old code, but this saves the container log as raw file using the default boefje raw file types, which I think is not what we want.

Reply:
I also think that's the case. I'll remove it

Reply:
Or do you mean that we should use other mime types?

Reply:
Update: the scheduler needs the mime-types to start with `error/`, so this will not work in general. If we only save the error logs when the task is still running we could just use the `error/boefje` mime type here now. Follow up ticket: https://github.com/minvws/nl-kat-coordination/issues/2728